SCF/SCF-assignment-9/Square.java

```
@@ -0,0 +1,129 @@
+package GETA2021.SCF.SCFAssignment9;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+/**
+ * Creates shape Square and implement different functions like
+ * getTtimeStamp,getShapeType,getPerimeter,getOrigin getArea,isPointEnclosed
+ * 
+ * @since 03-03-2021
+ * @author Hitesh Pareek
+ *
+ */
+public class Square implements Shape {
+	final String shapeType = Shape.shapeType.Square.toString();
+	double side;
+	Point origin;
+	Timestamp timeStamp;
+
+	/**
+	 * Constructor to create Square
+	 * 
+	 * @param origin
+	 * @param side
+	 */
+	Square(Point origin, double side) {
+		this.side = side;
+		this.origin = origin;
+		this.timeStamp =new Timestamp(new Date().getTime());
```

Constructor should throw exceptions for invalid parameters.
This comment applies to all the class created above.